### ISSUE TITLE
TypeError: Cannot read property 'type' of null

### ISSUE THREAD

**Issue by zicodeng** [FormidableLabs/webpack-dashboard#335]
> Please provide a description of the bug / issue, and provide the details below:

Not sure if this is related to this plugin or not, but after upgrading `webpack-dev-server` to the latest version (`4.0.0`), this error started showing up. However, `webpack-dev-server` runs fine without this plugin 🤔 

```
TypeError: Cannot read property 'type' of null
    at Socket.<anonymous> (/Users/zicodeng/code/wenwentips/client/node_modules/webpack-dashboard/bin/webpack-dashboard.js:79:21)
    at Socket.emit (events.js:315:20)
    at Socket.emitUntyped (/Users/zicodeng/code/wenwentips/client/node_modules/socket.io/dist/typed-events.js:69:22)
    at /Users/zicodeng/code/wenwentips/client/node_modules/socket.io/dist/socket.js:428:39
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
error Command failed with exit code 1.
```

====================================================================

###### If the issue is visual, please provide screenshots here

====================================================================

###### Steps to reproduce the problem

====================================================================

###### Please provide a gist of relevant files

1. package.json (specifically the script you are using to start the dashboard)

```
"webpack-cli": "^4.8.0",
"webpack-dashboard": "^3.3.3",
"webpack-dev-server": "^4.0.0",
```

2. webpack.config.js
3. index.js (Your express based dev server, if applicable)

====================================================================

###### More Details

- What operating system are you on? `MacOS`
- What terminal application are you using? `iTerm2`
- What version of webpack-dashboard are you using? `3.3.3`
- What is the output of running `echo $TERM`? `xterm-256color`

**Comment by zicodeng** [comment]
@ryan-roemer Thanks for your response! I have created a [repo](https://github.com/zicodeng/webpack-dashboard-issue-335-repro) for reproduction. Let me know if you have any questions. Thanks for looking into this :)

**Comment by zicodeng** [comment]
@ryan-roemer Any update?

**Comment by ryan-roemer** [comment]
Hi! Thanks for the repo. I’m on leave until next Tuesday, so I’ll likely turn to this next week. Thanks!

**Comment by ryan-roemer** [comment]
Confirmed. Working on a fix!

**Comment by ryan-roemer** [comment]
Fixed in `webpack-dashboard@3.3.6`. Thanks for repo @zicodeng !
